portfolio/src/main/webapp/blogStyle.css

```diff
@@ -0,0 +1,245 @@
+html {
+    scroll-behavior: smooth
+  }
```

Comment:
Looks like this curly brace got indented, and now the rest of the rules are indented.